go.mod

```diff
@@ -71,6 +71,7 @@ require (
 	github.com/gravitational/oxy v0.0.0-20221029012416-9fbf4c444680
 	github.com/gravitational/reporting v0.0.0-20210923183620-237377721140
 	github.com/gravitational/roundtrip v1.0.2
+	github.com/gravitational/teleport-plugins v0.0.0-20230306172745-5f49efae731f
```

Comment:
All other changes are automatic via `go mod tidy`, due to plugins pulling in newer versions. If we want to prevent those upgrades, we must downgrade in plugins.

Reply:
Shall we downgrade (some or all of those) in `teleport-plugins`? The situation is a bit weird, since plugins v12 are released from their `master` branch, while for teleport we maintain `branch/v12` etc.

I think making `teleport-plugins` go.mod versions to not exceed `teleport` `branch/v12` would make the most sense if we're going this way.

Reply:
How are we supposed to do breaking changes in `api/` from one major version to the next, if `teleport-plugins` is not versioned by release branches? How are we going to keep the dependency on `teleport-plugins` up to date for the release branches other than the last major one for the duration of support?

Reply:
These are some very good points.

I'm not aware of any progress of moving `teleport-plugins` into `teleport` so far.

Perhaps moving [access/slack](https://github.com/gravitational/teleport-plugins/tree/master/access/slack) (but not `cmd/slack`) here could be a first step. Then we can make `teleport-plugins/.../cmd/slack` depend on `teleport` (same version as `teleport/api`), and the plugin itself can evolve in this repo (tied to `api` major version). Meanwhile leaving `cmd/slack` in `teleport-plugins` means we do not have to overhaul entire CI in the scope of this project.

/cc @r0mant for thoughts.